### PR TITLE
Normalize the app name

### PR DESCRIPTION
**Pull request by mattford63**
**Normalize the app name**
- will fix alerting on witan.gateway

### CODE REVIEW

File: deployment/production-marathon.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
       ]
     }
   },
-  "id": "witan-gateway",
+  "id": "OVERRIDDEN_BY_JENKINS",
```
Comment (acron0) [inline deployment/production-marathon.json]: I thought this was used in the service route thing?

Comment (mattford63) [reply deployment/production-marathon.json]: This makes in consistent with how datastore does it.

Comment (mattford63) [reply deployment/production-marathon.json]: Also the `.`s appear to be re-written by DCOS when creating the internal DNS to hyphens.

Comment (acron0) [reply deployment/production-marathon.json]: Ah, ok

**Pull request merged**